topology/nhlt/intel/dmic/dmic-process.c

```diff
@@ -0,0 +1,1365 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I wonder if it would make it easier if you split the dmic-process.* files into a separate commit. There are pretty much exactly copy from the SOF code right?

Reply:
I mean whatever makes things easier to read. I've already done 2 iterations from comments to back and forth with single commits to multiple commits :) but yeah as it is not so clear -> there's probably issue in the patch division